types/redis/index.d.ts

```diff
@@ -32,30 +32,182 @@ export interface RetryStrategyOptions {
 
 export type RetryStrategy = (options: RetryStrategyOptions) => number | Error | unknown;
 
+/**
+ * Client options.
+ * @see https://github.com/NodeRedis/node-redis#user-content-options-object-properties
+ */
 export interface ClientOpts {
+    /**
+     * IP address of the Redis server.
+     * @default 127.0.0.1
+     */
     host?: string;
+    /**
+     * Port of the Redis server.
+     * @default 6379
+     */
     port?: number;
+    /**
+     * The UNIX socket string of the Redis server.
+     * @default null
+     */
     path?: string;
+    /**
+     * The URL of the Redis server.\
+     * Format:
+     * [redis[s]:]//[[user][:password@]][host][:port][/db-number][?db=db-number[&password=bar[&option=value]]]\
+     * More info avaliable at [IANA](http://www.iana.org/assignments/uri-schemes/prov/redis).
+     * @default null
+     */
     url?: string;
     parser?: string;
+    /**
+     * Set to `true`, Node Redis will return Redis number values as Strings instead of javascript Numbers.
+     * Useful if you need to handle big numbers (above `Number.MAX_SAFE_INTEGER` === 2^53).
+     * Hiredis is incapable of this behavior, so setting this option to `true`
+     * will result in the built-in javascript parser being used no matter
+     * the value of the `parser` option.
+     * @default null
+     */
     string_numbers?: boolean;
+    /**
+     * If set to `true`, then all replies will be sent to callbacks as Buffers instead of Strings.
+     * @default false
+     */
     return_buffers?: boolean;
+    /**
+     * If set to `true`, then replies will be sent to callbacks as Buffers.
+     * This option lets you switch between Buffers and Strings on a per-command basis,
+     * whereas `return_buffers` applies to every command on a client.\
+     * **Note**: This doesn't work properly with the pubsub mode.
+     * A subscriber has to either always return Strings or Buffers.
+     * @default false
+     */
     detect_buffers?: boolean;
+    /**
+     * If set to `true`, the keep-alive functionality is enabled on the underlying socket.
+     * @default true
+     */
     socket_keepalive?: boolean;
+    /**
+     * Initial Delay in milliseconds,
+     * and this will also behave the interval keep alive message sending to Redis.
```

Comment:
"this will also behave the interval" — I think there's a typo here?

Reply:
I copied this from [README](https://github.com/NodeRedis/node-redis/blob/master/README.md).
It looks like a typo. I am new to redis and I don't know how to write it correctly. The only thing I can do, is to simplify the documentation for this option. I can leave it to *Initial Delay in milliseconds.*

Please tell me if you agree to simplify it.

Reply:
> This will also set the initial delay for keep-alive packets being sent to Redis.

Or something like that would be fine I think. Bad code review on their end it seems, we can do better 😂

Reply:
^ yeah, this would work.

(I figured it was probably a mistake in the source but wanted ours to make sense 😜)

Reply:
I added **This will also set the initial delay for keep-alive packets being sent to Redis.** and pushed the change.